AWS/AWS_Setup.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+#This script is intended to automatically configure new EC2 instances on launch. To make use of it; copy the contents to your clipboard, log-in to the AWS console, select EC2, and click on 
+#the "launch template" option in the left-hand navigation pane. Make sure you are using the Ubuntu 18.04 AMI (with a t2.micro instance if you want to stay within the bounds of free tier), and scroll down
+#to advanced details. Click the dropdown, and paste the contents of this file you previously copied into user data, and click "create template version." You will now be able to launch new instances with
+#this template via the launch instances drop down on the EC2 dashboard.
+IMAGE="<image name>"
+#Install Docker
+(
+set -e
+curl -fsSL https://get.docker.com -o get-docker.sh
+sh get-docker.sh
+docker pull "$IMAGE"
+set +e
+)
+
+#Create .env, edit as necessary
+mkdir /home/ubuntu/docker_config && cat > /home/ubuntu/docker_config/.env <<EOF
+LOCAL_DATA=/app/data
+DATASTORE_NAME=S3
+DATASTORE_PATH=performance-tracker-test
+TMP_DIR=/app/data/tmp
+METRO_LINES=801,802,803,804,805,806
+METRO_AGENCY=lametro-rail
+TIMEZONE=America/Los_Angeles
+SCHEDULE_URL=https://gitlab.com/LACMTA/gtfs_rail/raw/master/gtfs_rail.zip
+VEHICLE_API_URL=http://webservices.nextbus.com/service/publicJSONFeed
+LOG_TIMESTAMPS=TRUE
+AWS_ACCESS_KEY_ID=<access key id>
+AWS_SECRET_ACCESS_KEY=<secret key>
```

Comment:
possibly these could be extracted from a .env file (passed in by the path as a command line arg), that way the user wouldn't have to modify this script to get it to run. Just validate at the beginning that a valid .env exists. I prefer the idea of a .env file easily accessible to the user so that it can be easily modified at run time.

Also for AWS keys, if we set up the EC2 instance correctly then the permissions to access S3 should be at the IAM role level assigned to the instance. Then these env variables are not required. But that does add complexity to setup, which is obviously what this script is intending to reduce. It's more secure to use AWS IAM roles, but we can look at that further down the track.